arches_component_lab/src/arches_component_lab/widgets/NonLocalizedTextWidget/components/NonLocalizedTextWidgetEditor.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import InputText from "primevue/inputtext";
+import GenericFormField from "@/arches_component_lab/generic/GenericFormField.vue";
+
+import type { NodeData } from "@/arches_component_lab/types.ts";
+
+const { nodeAlias, value } = defineProps<{
```

Comment:
I also don't know that I love the prop being named `value`, it can be confusing in more complex widgets but I don't have a better name offhand 😬

Reply:
`nodeValue` maybe?